MediaPlayer/MediaPlayer.js

```diff
@@ -536,6 +536,17 @@ const MediaPlayerBehaviorDecorator = hoc((config, Wrapped) => { // eslint-disabl
 			}
 		};
 
+		asyncPlay = () => {
+			const playPromise = this.play();
+			if (playPromise) {
+				playPromise.then(() => {
+					this.play();
```

Comment:
I wonder this line is needed.
I saw https://developers.google.com/web/updates/2017/06/play-request-was-interrupted and when i tried without this line it looks work.

If you don't need it, how about just modifying the play function without creating a separate asyncPlay function?

Reply:
You are right, I didn't need it. Thank you, I pushed the change.